packages/okta-auth-js/lib/TokenManager.js

```diff
@@ -228,7 +227,10 @@ function TokenManager(sdk, options) {
         storageProvider = sessionStorage;
         break;
       case 'cookie':
-        storageProvider = storageUtil.getCookieStorage(options);
+        storageProvider = storageUtil.getCookieStorage({
+          secure: sdk.options.secureCookies,
+          sameSite: 'strict' // token storage should only be accessed by javascript
```

Comment:
I think the comment here is misleading.  It _is_ an explanation of why 'strict' is acceptable, but it _looks_ like a (wrong) explanation of what 'strict' does.